Sources/AngouriMath/Convenience/MathS.cs

```diff
@@ -888,7 +888,12 @@ public static Matrix Vector(params Entity[] values)
         /// </summary>
         public static partial class Settings
         {
-
+            [ConstantField]
+            private static bool explicitParsingOnly=false;
+            /// <summary>
+            /// Determine if it should only parse if it is explicit
+            /// </summary>
+            public static bool ExplicitParsingOnly { get => explicitParsingOnly;}
```

Comment:
This thing should be a `Setting<T>`. For reference, see
```cs
public static Setting<bool> DowncastingEnabled => downcastingEnabled ??= true;
[ThreadStatic] private static Setting<bool>? downcastingEnabled;
```
Same way, you can do for `ExplicitParsingOnly`

Reply:
Okay ,im changing it soon